data_validation/cli_tools.py

```diff
@@ -291,6 +291,12 @@ def _configure_run_parser(subparsers):
         "-filters",
         help="Filters in the format source_filter:target_filter",
     )
+    run_parser.add_argument(
+        "--format",
+        "-format",
+        default="table",
+        help="Set the format for printing command output",
```

Comment:
In the 'help' field, we can also include the supported format options [table, text, csv, json]

Reply:
Done.